lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/validation/JarValidator.java

```diff
@@ -87,6 +89,46 @@ public JWTClaimsSet validateRequestJwt(SignedJWT signedJWT, String clientId)
         }
     }
 
+    private void validateScope(String clientId, JWTClaimsSet claimsSet)
+            throws JarValidationException {
+        try {
+            String scope = claimsSet.getStringClaim("scope");
+            var validScopes = configService.getValidClientScopes(clientId);
+            if (validScopes.isEmpty() || !validScopes.contains(scope)) {
```

Comment:
I think there's another case here worth checking. We would never expect to receive `openid` and `reverification` in the scope - it can be a space separated (I think) list of scopes.